packages/astro/src/@types/astro.ts

```diff
@@ -1438,7 +1438,48 @@ export interface AstroUserConfig {
 		 * @name markdown.shikiConfig
 		 * @typeraw {Partial<ShikiConfig>}
 		 * @description
-		 * Shiki configuration options. See [the Markdown configuration docs](https://docs.astro.build/en/guides/markdown-content/#shiki-configuration) for usage.
+		 * Shiki is our default syntax highlighter. You can configure all options via the `markdown.shikiConfig` object:
+	   * 
+		 * ```js title="astro.config.mjs"
+	   * import { defineConfig } from 'astro/config';
+		 * 
+	   * export default defineConfig({
+		 *   markdown: {
+	   *     shikiConfig: {
+		 *       // Choose from Shiki's built-in themes (or add your own)
+     *       // https://shiki.style/themes
```

Comment:
uh, it doesn't look like that for me??

![image](https://github.com/user-attachments/assets/edcd7930-03b9-4dcb-bf12-136b7985a987)

Reply:
Mind you, the spaces and tabs *are* terrible, but it does not look uneven to me!

Reply:
Yeah, I thought it was related to spaces and tabs... but we don't see the same thing at all 😅 

![astro-formatting](https://github.com/user-attachments/assets/8ee0a24f-b1b7-4d66-82a7-0c2f1499263a)

But I don't see spaces and tabs either... so the display must be different because of that.

Reply:
@ArmandPhilippot that's a GitHub thing, there's a setting to tell GitHub to show you whitespaces